### PR TITLE
Add Chinese (Traditional, Taiwan) localization

### PR DESCRIPTION
**Pull request by stiiveo**

### CODE REVIEW

**Comment by sindresorhus** [comment]
@hank121314 Would you be willing to review this?

File: Sources/KeyboardShortcuts/Localization/zh-TW.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"record_shortcut" = "記錄快捷鍵";
+"press_shortcut" = "鍵入快捷鍵";
+"keyboard_shortcut_used_by_menu_item" = "此快捷鍵無法使用，因為它已被選單項目「%@」使用。";
+"keyboard_shortcut_used_by_system" = "此快捷鍵無法使用，因為它已是系統通用快捷鍵。";
+"keyboard_shortcuts_can_be_changed" = "可以在「系統偏好設置 › 鍵盤 › 快速鍵」中更改大多數的系統快捷鍵。";
```
Comment (hank121314) [inline Sources/KeyboardShortcuts/Localization/zh-TW.lproj/Localizable.strings:5]:
```suggestion
"keyboard_shortcuts_can_be_changed" = "可以在「系統偏好設定 › 鍵盤 › 快速鍵」中更改大多數的系統快捷鍵。";
```

**Review by hank121314 — CHANGES_REQUESTED** [review]
Except for a typo. LGTM!

File: Sources/KeyboardShortcuts/Localization/zh-TW.lproj/Localizable.strings (hunk 1 above)
Comment (stiiveo) [reply Sources/KeyboardShortcuts/Localization/zh-TW.lproj/Localizable.strings:5]: Thank you @hank121314 for spotting the typo. 
Appreciate it!

**Comment by sindresorhus** [comment]
Shouldn't the language code be `zh-Hant`? Because `zh-Hant` matches zh-Hant, zh-Hant-TW, zh-Hant-HK, while `zh-TW` only matches `zh-TW`.

**Comment by stiiveo** [comment]
`zh-TW` is a locale code while `zh-Hant` refers to general Traditional Chinese. 
If you are to make the localization specific to which locale of the language, I believe `zh-TW` or `zh-Hant-TW` should be chosen.

**Comment by sindresorhus** [comment]
I'm probably incorrectly assuming that `zh-TW` and `zh-Hant` would be the same thing text-wise.

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/KeyboardShortcuts/releases/tag/v1.3.0

**Pull request merged**